qtawesome/__init__.py

```diff
@@ -222,3 +223,49 @@ def set_defaults(**kwargs):
     """
     return set_global_defaults(**kwargs)
 
+
+class IconWidget(QtWidgets.QLabel):
+    '''
```

Comment:
@doudz, please use double quotes (`"`) instead of single ones (`'`) for all docstrings in this class.